Artifacts/PgSql/Structure/Admin/0131-Add-ApiClientOdsInstances-Table-Modify-GetClientForToken.sql

```diff
@@ -0,0 +1,87 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+ALTER TABLE dbo.OdsInstances
+    ADD ConnectionString TEXT;
+
+CREATE TABLE dbo.ApiClientOdsInstances
+(
+    ApiClient_ApiClientId INT NOT NULL,
+    OdsInstance_OdsInstanceId INT NOT NULL,
+    PRIMARY KEY(ApiClient_ApiClientId, OdsInstance_OdsInstanceId)
+);
+
+ALTER TABLE dbo.ApiClientOdsInstances
+    ADD CONSTRAINT FK_ApiClientOdsInstances_ApiClient FOREIGN KEY (ApiClient_ApiClientId) 
+    REFERENCES dbo.ApiClients(ApiClientId)
+    ON DELETE CASCADE;
+
+ALTER TABLE dbo.ApiClientOdsInstances
+    ADD CONSTRAINT FK_ApiClientOdsInstances_OdsInstance FOREIGN KEY (OdsInstance_OdsInstanceId) 
+    REFERENCES dbo.OdsInstances(OdsInstanceId)
+    ON DELETE CASCADE;
```

Comment:
`ON DELETE CASCADE` here isn't present in the SQL Server script. We should try to keep the database structures logically in sync, and stylistically consistent with the rest of the model. Are cascading deletes used on other FK constraints in this database as well?

Reply:
Agreed, I missed that. The other admin tables don’t cascade delete.  @simpat-adam, can you change these statements to remove the cascade deletes and make them consistent with the other scripts such as https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/af95cd3459f4895ac610e421987a1f685716cec2/Artifacts/MsSql/Structure/Admin/0020-Tables.sql. I reopened the branch for this PR so you should be able to push to it.

Reply:
@simpat-adam , It looks like you may have to do a new PR

Reply:
@mjaksn I saw that we do cascade deletes on ApiClientOwnershipTokens. I am modifying those scripts in 5790 ticket so I will remove the cascade deletes on that PR.

Reply:
@simpat-adam, sounds good. I'll close 5786